recipes-core/torizon-conf/torizon-conf/containers-tags.sh

```diff
@@ -1,38 +1,21 @@
 # List of containers tags Torizon is compatible with
 
-export CT_TAG_DEBIAN="3-bookworm"
-export CT_TAG_ASPDOTNET_6="3-6.0"
-export CT_TAG_ASPDOTNET_6_DEBUG="3-6.0"
-export CT_TAG_ASPDOTNET_8="3-8.0"
-export CT_TAG_ASPDOTNET_8_DEBUG="3-8.0"
-export CT_TAG_CHROMIUM="3"
-export CT_TAG_COG="3"
-export CT_TAG_DOTNET_6="3-6.0"
-export CT_TAG_DOTNET_6_DEBUG="3-6.0"
-export CT_TAG_DOTNET_6_UNO="3-6.0"
-export CT_TAG_DOTNET_6_UNO_DEBUG="3-6.0"
-export CT_TAG_DOTNET_8="3-8.0"
-export CT_TAG_DOTNET_8_DEBUG="3-8.0"
-export CT_TAG_DOTNET_8_UNO="3-8.0"
-export CT_TAG_DOTNET_8_UNO_DEBUG="3-8.0"
-export CT_TAG_GRAPHICS_TESTS="3"
-export CT_TAG_GRAPHICS_TESTS_VIVANTE="3"
-export CT_TAG_PORTAINER_DEMO="3"
-export CT_TAG_QT5_WAYLAND="3"
-export CT_TAG_QT5_WAYLAND_EXAMPLES="3"
-export CT_TAG_QT5_WAYLAND_EXAMPLES_VIVANTE="3"
-export CT_TAG_QT5_WAYLAND_VIVANTE="3"
-export CT_TAG_QT6_WAYLAND="3"
-export CT_TAG_QT6_WAYLAND_EXAMPLES="3"
-export CT_TAG_QT6_WAYLAND_EXAMPLES_VIVANTE="3"
-export CT_TAG_QT6_WAYLAND_VIVANTE="3"
-export CT_TAG_RT_TESTS="3"
-export CT_TAG_STRESS_TESTS="3"
-export CT_TAG_TORIZONCORE_BUILDER="3"
-export CT_TAG_WAYLAND_BASE="3"
-export CT_TAG_WAYLAND_BASE_VIVANTE="3"
-export CT_TAG_WESTON="3"
-export CT_TAG_WESTON_TOUCH_CALIBRATOR="3"
-export CT_TAG_WESTON_VIVANTE="3"
-export CT_TAG_QT6_WAYLAND="3"
-export CT_TAG_QT6_WAYLAND_EXAMPLES="3"
+export CT_TAG_ASPDOTNET_8="4"
+export CT_TAG_CHROMIUM="4"
+export CT_TAG_COG="4"
+export CT_TAG_DEBIAN="4"
+export CT_TAG_DOTNET_8="4"
+export CT_TAG_DOTNET_8_DEBUG="4"
+export CT_TAG_DOTNET_8_GTK_3="4"
+export CT_TAG_DOTNET_8_WAYLAND="4"
+export CT_TAG_GRAPHICS_TESTS="4"
+export CT_TAG_QT5_WAYLAND="4"
+export CT_TAG_QT5_WAYLAND_EXAMPLES="4"
+export CT_TAG_QT6_WAYLAND="4"
+export CT_TAG_QT6_WAYLAND_EXAMPLES="4"
```

Comment:
@leonheldattoradex remove for now?

Reply:
It's fine to leave it in, we'll release shortly after.